### PR TITLE
Delete contact

### PR DESCRIPTION
**Pull request by M-OC**
Contacts created in 'settings' can now be deleted.

### CODE REVIEW

File: Client/Src/actions/emergency-contact-actions.js [hunk 1]
```diff
@@ -83,7 +83,19 @@ export const addEmergencyContactSuccess = (payload) => {
...
-  console.log('Got to RemoveEmergencyContact Action', payload);// This needs to use the index to remove from state. Send the contact, with userId to server for removal
+  return (dispatch) => {
+    fetch('http://localhost:8000/user/' + payload.user_id + '/contacts',
+    {
+      method: 'DELETE',
+      headers: {'Content-Type': 'application/json'},
+      body: JSON.stringify(payload)
+    })
+    .then((response) => {
+      console.log("Server resonse: ", response);
```
Comment (urbantumbleweed) [inline Client/Src/actions/emergency-contact-actions.js]: Can you please delete console.log

File: Client/Src/components/emergency-contact-list-item.js [hunk 2]
```diff
@@ -17,15 +18,24 @@ export default class EmergencyContactListItem extends Component {
   }
 
   render() {
-    const { contact, id, actions } = this.props;
+    //style={styles.userContactDeleteButton}
+    const { key, contact, user_id, id, actions, contactList } = this.props;
```
Comment (urbantumbleweed) [inline Client/Src/components/emergency-contact-list-item.js:22]: is `user_id` and `id` the same?  What are they referring to?

File: Server/Routes/routes.js [hunk 3]
```diff
@@ -49,9 +49,14 @@ module.exports = function (app, express) {
 
 // UPDATE USER CONTACTS
   app.put('/user/:user_id/contacts', function (req, res) {
+  		console.log('Added!');
```
Comment (urbantumbleweed) [inline Server/Routes/routes.js]: If this works, please remove console.log

File: Client/Src/components/emergency-contact-list-item.js (hunk 2 above)
Comment (M-OC) [inline Client/Src/components/emergency-contact-list-item.js:22]: user_id refers to the user. id refers to the contact's id.

**Pull request merged**